overwatch-rs/src/overwatch/handle.rs

```diff
@@ -10,6 +10,31 @@ use tracing::{error, info, instrument};
 
 // internal
 use crate::services::relay::Relay;
+use crate::BoxFuture;
```

Comment:
Because if using `async_trait`, then if use vscode, we will miss the code snippet and auto-add missing methods functionalities, and `BoxFuture` is equal to the expanded code of async_trait.

Reply:
uhm, that is pretty weird. I wouldn't make a decision just based in a tooling failure in this case. Please use `async_trait` as I think it exposed clearly how the methods should be implemented. Also at some point when the feature arrives we can just use the async methods in traits without the macro.